src/django_markup/fields.py

```diff
@@ -13,7 +13,7 @@ def __init__(self, default=False, formatter=formatter, *args, **kwargs):
         if default:
             if default not in formatter.filter_list:
                 raise ImproperlyConfigured("'%s' is not a registered markup filter. Registered filters are: %s." %
-                                           (default, ', '.join(formatter.filter_list.iterkeys())))
+                                           (default, ', '.join(formatter.filter_list.keys())))
```

Comment:
It's been a while, but I'm pretty sure this will work on both 2 & 3. IIRC, the difference is that it will return a list in 2 and an iterator in 3. Both should behave the same in this scenario.